src/components/Breeds.js

```diff
@@ -1,27 +1,62 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import "./Breeds.css";
 
-class Breeds extends React.Component {
-  render() {
-    return (
-      <div className="Breeds">
-        <h2 className="Breeds-title">Select a Breed</h2>
-        <p>
-          <select className="Breeds-select">
-            <option>beagle</option>
-            <option>bluetick</option>
-            <option>bullterrier-staffordshire</option>
-            <option>malinois</option>
-            <option>wolfhound-irish</option>
-          </select>
-        </p>
-        <img className="Breeds-image" src="http://via.placeholder.com/300x300" />
-        <p>
-          <button className="Breeds-button">Show me more!</button>
-        </p>
-      </div>
-    );
+const Breeds = () => {
+  const options = [
+    "beagle",
+    "bluetick",
+    "bullterrier/staffordshire",
+    "malinois",
+    "wolfhound/irish"
+  ];
+
+  const [selectedPhoto, setSelectedPhoto] = useState(null);
+  const [selectState, setSelectState] = useState(options[0]);
```

Comment:
Perhaps, we could use `selectBreed` and `setSelectedBreed` to show what state is being held here.

Reply:
Thanks Laura, i changed the names of state